core/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![no_std]
```

Comment:
```suggestion
#![cfg_attr(not(feature = "std"), no_std)]
```

Reply:
We don't want to enable `no_std` when the `std` feature is enabled as with all crates.

Reply:
indeed